plutus-core/plutus-ir/test/PlutusIR/Transform/RewriteRules/trace/unConstrConstrDataFst

```diff
@@ -0,0 +1 @@
+/home/bezirg/repos/plutus/plutus-core/plutus-ir/test/PlutusIR/Transform/RewriteRules/pir/unConstrConstrDataFst
```

Comment:
a symbolic link if you don't mind

Reply:
@kwxm Actually good that you checked it, I forgot to make the symlink a relative one